README.md

```diff
@@ -42,6 +66,37 @@ npm run start:dev
  
 Server API is listening on [`http://localhost:7080`](http://localhost:7080)
 
+### Database
+
+Before connecting to the RDS, you must SSH into a Cosmos component that has been granted permission to communicate with the RDS in its policy. Run:
+
+```
+ssh -f -L [port]:[rds name]:[port] [ssh] -N
+```
+#### Migrations
+First, make sure you've installed Knex.js globally:
```

Comment:
You can also run it via `npx`: this uses the executable in your node_modules. That's what I've been doing :)